Philipona/workshop feedback (#132)
acend/helm-basics-training: content/en/docs/02/_index.md

helm ls --namespace <namespace>
```

```
NAME            NAMESPACE       REVISION  UPDATED                                   STATUS    CHART        APP VERSION
myfirstrelease  <namespace>     1         2021-04-14 14:29:58.808282266 +0200 CEST  deployed  mychart-0.1.01.16.0  
```

> ## Task {{% param sectionnumber %}}.3: Expose Application

Reviewer: ```suggestion
## Task {{% param sectionnumber %}}.3: Expose the application
```

REVISION: 2
NOTES:
1. Get the application URL by running these commands:
  http://mychart-<namespace>.<appdomain>/
```

{{% onlyWhenNot mobi %}}
Check whether the ingress was successfully deployed by accessing the URL `http://mychart-<namespace>.<appdomain>/`

{{% /onlyWhenNot %}}
{{% onlyWhen mobi %}}

> Check whether the ingress was successfully deployed by accessing the URL `https://mychart-<namespace>.<appdomain>/`

Reviewer: I'm not sure if this is the case with Mobi...

Update the replica count of your nginx Deployment to 2 using `--set name=value`

### Solution Task {{% param sectionnumber %}}.4

```bash
helm upgrade --namespace <namespace> --set replicaCount=2 myfirstrelease ./mychart
```

Values that have been set using `--set` can be reset by helm upgrade with `--reset-values`.

Verify the replicaCount with the following command:

```bash
{{% param cliToolName %}} get pods --namespace <namespace>
```

## Task {{% param sectionnumber %}}.5: Rollback a release

Helm also provides the functionality to roll back a release to a specific revision number.
In the previous tasks, you might have noticed the `REVISION` be increased each time you installed or updated a release.

> Every change you make to a release by installing or upgrading it, will increase the `REVISION`. The actual deployed revision can be displayed with the following command:

Reviewer: ```suggestion
Every change you make to a release by installing or upgrading it will increase the `REVISION`. The actual deployed revision can be displayed with the following command:
```

```bash
helm ls --namespace <namespace>
```

```
NAME            NAMESPACE       REVISION  UPDATED                                   STATUS    CHART        APP VERSION
myfirstrelease  <namespace>     3         2021-04-14 14:29:58.808282266 +0200 CEST  deployed  mychart-0.1.01.16.0  
```

Let's now rollback our release `myfirstrelease` to revision 2.

```bash
helm rollback myfirstrelease 2 --namespace <namespace>
```

> The replicaCount should be back down to 1 after the rollback. Check that with the following command:

Reviewer: ```suggestion
The replicaCount should be back down to 1 after the rollback. Check if that's true with the following command:
```
